bundle/artifacts/artifacts.go

```diff
@@ -186,6 +186,9 @@ func rewriteArtifactPath(b *bundle.Bundle, f *config.ArtifactFile, job *resource
 			if lib.Jar != "" && isArtifactMatchLibrary(f, lib.Jar, b) {
 				lib.Jar = remotePath
 			}
+			if lib.Requirements != "" && isArtifactMatchLibrary(f, lib.Requirements, b) {
```

Comment:
You don't need to call `isArtifactMatchLibrary`, it won't work anyway because it checks that libraries defined in `artifacts` sections matches the ones used in library and it's not relevant for requirements.txt

Reply:
Done, please review